lib/core/src/lib/pipes/pipe.module.ts

```diff
@@ -36,6 +36,7 @@ import { MomentDateTimePipe } from './moment-datetime.pipe';
 import { FilterStringPipe } from './filter-string.pipe';
 import { FilterOutArrayObjectsByPropPipe } from './filter-out-every-object-by-prop.pipe';
 import { DateTimePipe } from './date-time.pipe';
+import { DatePipe } from './date.pipe';
```

Comment:
should be AdfDatePipe not to clash with Angular DatePipe

Reply:
renamed